docs/DeveloperGuide.md

```diff
@@ -118,52 +154,99 @@ How the `Logic` component works:
-
+<div style="width:80%;margin:0">
+    <b>Figure 1.8</b> Class diagram of 'Model'
+</div>
+<br>
 
 The `Model` component,
 
-* stores the address book data i.e., all `Event` objects (which are contained in a `UniqueEventList` object).
+* stores the event book data i.e., all `Event` objects (which are contained in a `UniqueEventList` object).
 * stores the currently 'selected' `Event` objects (e.g., results of a search query) as a separate _filtered_ list which is exposed to outsiders as an unmodifiable `ObservableList<Event>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Event` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Event` needing their own `Tag` objects.<br>
-
-<img src="images/BetterModelClassDiagram.png" width="450" />
+### Event component
+**API** : [`Event.java`](https://github.com/AY2223S2-CS2103T-T11-3/tp/blob/master/src/main/java/seedu/address/model/event/Event.java)
 
+<img src="images/EventClassDiagram.png" width="450" />
+<div style="width:80%;margin:0">
+    <b>Figure 1.9</b> Class diagram of 'Event'
+</div>
+<br>
+
+The `Event` component,
+
+* stores each category of information of an event in individual attributes and each attribute is an object.
+* arrtibutes are:
+   * `Address`
+   * `Mark`
+   * `Name`
+   * `Tag`
+   * `Rate`
+   * `Time`
+   * `Contact`
+* when an event is created the above objects are created using user inputs and then the event object will be created.
+* the event will be added to the UniqueEventList.
+
+### Contact component
+**API** : [`Contact.java`](https://github.com/AY2223S2-CS2103T-T11-3/tp/blob/master/src/main/java/seedu/address/model/contact/Contact.java)
+
+<img src="images/ContactClassDiagram.png" width="450" />
+<div style="width:80%;margin:0">
+    <b>Figure 2.0</b> Class diagram of 'Contact'
 </div>
+<br>
+
+The `Contact` component,
+- it is made up of 2 objects the `ContactName` and `ContactPhone`
+- both objects are created using user input.
+
+[//]: # (<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative &#40;arguably, a more OOP&#41; model is given below. It has a `Tag` list in the `AddressBook`, which `Event` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Event` needing their own `Tag` objects.<br>)
```

Comment:
Should `AddressBook` be `EventBook` instead?

Reply:
These lines are commented out so they were not really edited. It was left there incase group decides to include that portion in DG, then it will be edited, else it will be deleted in time